pkg/proto/datadog/trace/stats.proto

```diff
@@ -62,7 +62,7 @@ message ClientStatsBucket {
 	int64 agentTimeShift = 4;
 }
 
-enum TraceRootFlag {
+enum Trilean {
```

Comment:
I had to look up the name `Trilean` as I was totally unfamiliar with this term, is this a term that already exists in the backend or in other areas? If not perhaps a better name might be something like OptionalBoolean?

Reply:
We're already using `Trilean` in the backend